### ISSUE TITLE
Add Exception Handling

### ISSUE THREAD

**Issue by VickiLanger** [VickiLanger/code-questions-bot#8]
Need to add exception handling to tweet.py

```python
try:
    api.update_status(question)
except:
    # not sure what would go here?
finally:
    api.update_status('I seem to not be working properly. Bug @Vicki_Langer about getting me fixed or help her out by submitting a pull request https://github.com/VickiLanger/code-questions-bot')
```

---
*Don't forget to follow the bot on [Twitter](https://www.twitter.com/CodeQBot) to see the hard work you put in.*

**Comment by sroy8091** [comment]
@VickiLanger we can send a report email in case of exception, and keep on restarting the server after a configurable amount of time.

**Comment by VickiLanger** [comment]
@sroy8091 Yes, an email would work. That makes sense. I don't see why we couldn't restart the server until it works.

**Comment by achoudh5** [comment]
@VickiLanger can we put the specific `status code` under except part? 

[Reference](https://lw.microstrategy.com/msdz/MSDL/GARelease_Current/docs/projects/RESTSDK/Content/topics/REST_API/REST_API_Handling_exceptions.htm)

**Comment by VickiLanger** [comment]
@achoudh5 that seems like it would make sense. Honestly, I have not done any exception handling outside of some practice. So, I'm not sure what it normally looks like.

**Comment by achoudh5** [comment]
If I may ask, what I understood by `tweet.py` and this error handling case is that after making sure the user enters their credentials right, it's taking some questions from a pool of questions and putting them as tweet right?

**Comment by VickiLanger** [comment]
Yes. `tweet.py` uses questions from files in `questions_dir`

**Comment by achoudh5** [comment]
Awesome! so, can we use an edge case file like if a user sends an `empty` tweet or some tweets they wants to avoid but since it's automated they don't have any control over it?

**Comment by VickiLanger** [comment]
Yes. That makes sense. An empty tweet definitely shouldn't happen.

**Comment by achoudh5** [comment]
Is that the only case you want to consider for now? I can create a PR for that if it's fine

**Comment by VickiLanger** [comment]
That's the only thing I can think of. Go for it
